package.json

```diff
@@ -18,5 +18,24 @@
     "eslint": "^6.6.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.18.2"
+  },
```

Comment:
These should not be added to the root project. Specify runtime dependencies at the module that uses them.

Reply:
added

Reply:
I expected these to NOT be in the root package. Why are they added here?